src/aas-api-webapp-repository/Startup.cs

```diff
@@ -19,13 +19,18 @@
 using Microsoft.OpenApi.Models;
 using Newtonsoft.Json.Converters;
 using Newtonsoft.Json.Serialization;
-using Swashbuckle.AspNetCore.Swagger;
-using Swashbuckle.AspNetCore.SwaggerGen;
 using AAS.API.Registry.Filters;
 using AAS.API.Registry.Models;
+using AAS.API.Repository;
+using AAS.API.Repository.ADTImpl;
```

Comment:
Basic idea was to have an implementation independent version here! But that's lost with these using statements!